### PR TITLE
Hide activities for categories

### PR DESCRIPTION
**Pull request by lukasz-kaniowski**
- hide 3 categories on the bottom by default
 - add show/hide button

### CODE REVIEW

File: app/assets/stylesheets/style.scss [hunk 1]
```diff
@@ -65,3 +65,24 @@
...
+  padding: 0;
+  margin: 0;
+  cursor: pointer;
+  background: transparent;
+  font: inherit;
+}
+
+
+.panel-indent {
+  border-left: 4px solid #bfc1c3;
```
Comment (nathancashmoredwp) [inline app/assets/stylesheets/style.scss]: worth replacing colours for variables... in this case $grey-2;

File: app/assets/stylesheets/style.scss [hunk 2]
```diff
@@ -65,3 +65,24 @@
...
+  padding: 10px 0 10px 15px;
```
Comment (nathancashmoredwp) [inline app/assets/stylesheets/style.scss]: Equivalent to:
padding: $gutter-one-third 0 $gutter-one-third $gutter-half;

File: app/views/sorted-activities.mustache [hunk 3]
```diff
@@ -28,74 +28,102 @@
...
+  <section data-test="category-READY" id="category-READY">
+    <h3 class="heading-small category-heading" data-test="category">{{#__}}category.READY.title{{/__}}</h3>
+    <span data-test="details">
     {{^perCategory.READY}}
       <p>{{#__}}sorted-activities.category-empty{{/__}}</p>
     {{/perCategory.READY}}
-    <ul>
-      {{#perCategory.READY}}
-        <li class="menu-container READY">
+      <ul>
```
Comment (nathancashmoredwp) [inline app/views/sorted-activities.mustache]: Indenting looks off when looking at the Raw view... worth checking ?

**Comment by lukasz-kaniowski** [comment]
feedback noted, have a look

**Comment by nathancashmoredwp** [comment]
:+1

**Comment by nathancashmoredwp** [comment]
👍

**Pull request merged**